tests/Bus/BusDispatcherTest.php

```diff
@@ -99,6 +139,18 @@ public function testOnConnectionOnJobWhenDispatching()
                 ],
             ]);
         });
+        $container->singleton('events', function () {
+            $mock = m::mock(\Illuminate\Events\Dispatcher::class);
```

Comment:
This is already imported?

Reply:
Not sure to follow, this the only 'events' mock in `testOnConnectionOnJobWhenDispatching` test.